Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -16,77 +19,116 @@ namespace Microsoft.MixedReality.Toolkit.SDK.UX.Pointers
     /// <summary>
     /// Base Pointer class for pointers that exist in the scene as GameObjects.
     /// </summary>
-    public abstract class BaseControllerPointer : ControllerPoseSynchronizer, IMixedRealityInputHandler, IMixedRealityPointer
+    [DisallowMultipleComponent]
```

Comment:
Does this mean we cannot have a pointer for each controller / hand in the scene?

Reply:
Do, it means we can only have a simple controller component on a single `GameObject`.

We can have as many pointers for as many hands as we'd like.